src/Microsoft.ML.Data/Transforms/NormalizeUtils.cs

```diff
@@ -0,0 +1,176 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.Model.Onnx;
+using Microsoft.ML.Runtime.Model.Pfa;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+[assembly: LoadableClass(typeof(void), typeof(Normalize), null, typeof(SignatureEntryPointModule), "Normalize")]
+
+namespace Microsoft.ML.Runtime.Data
+{
+    /// <summary>
+    /// Signature for a repository based loader of a IColumnFunction
+    /// </summary>
+    public delegate void SignatureLoadColumnFunction(ModelLoadContext ctx, IHost host, ColumnType typeSrc);
+
+    public interface IColumnFunctionBuilder
```

Comment:
>IColumnFunctionBuilder [](start = 21, length = 22)

I wonder if we can make all these interfaces internal. I know the idea was "hey we'll eventually have a bunch of transforms use this stuff" but as it is. They just sort of muddy the waters.